Automattic/pocket-casts-android
# Add database export option

## Description

This adds a database export menu option in the `Help & feedback` screen for debugging purposes.

## Testing Instructions

1. Open `Profile` tab
2. Tap on `Help & feedback`
3. Tap more option in the `Help & feedback` screen toolbar 
4. Tap `Export database`
5. Select an appropriate share option
6. Share the file
7. Download the file
8. Unzip the file
9. ✅ Notice that it includes: logs, shared prefs, and database files and they can be opened properly

## Screenshots or Screencast 

https://github.com/Automattic/pocket-casts-android/assets/1405144/5f64df08-39eb-4383-90bf-82ef088d85e5

File contents:
<img width="294" alt="file contents" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/7bf86004-df46-45e9-90f8-4309e31d09be">


## Checklist
 
#### I have tested any UI changes...
